microsoftgraph/msgraph-cli-archived
# Task/tests

## Overview

Adds **core** and **command module** tests

### Demo

N/A

### Notes

Tests in `msgraph.core.cli` are copied from `azure-cli` and modified to remove `azure-cli` specific implementations. Most of the `azure-cli` tests are left out because they either have a dependency on azure-cli extension support -- which we do not support -- or tests specific `azure-cli` implementations.

## Testing Instructions

* Pull these changes
* Activate a virtual environment `pipenv shell`
* Run `pytest`
* Observe that tests are executed.

Fixes #48, #46
